MapleServer2/Database/Classes/DatabaseQuest.cs

```diff
@@ -31,6 +32,12 @@ public long Insert(QuestStatus questStatus)
         foreach (dynamic data in results)
         {
             QuestStatus questStatus = (QuestStatus) ReadQuest(data);
+
+            if (questStatus == null)
```

Comment:
I think we moved to prefer `is null` vs `== null`, both here and in all other files